.circleci/config.yml

```diff
@@ -501,7 +501,7 @@ workflows:
       - integration-test-integration-airflow:
           matrix:
             parameters:
-              airflow-image: ['apache/airflow:2.1.3', 'apache/airflow:2.2.3']
+              airflow-image: ['apache/airflow:2.1.3-python3.7', 'apache/airflow:2.2.3-python3.7']
```

Comment:
The Airflow that customers will be using is now [2.2.4](https://airflow.apache.org/docs/apache-airflow/stable/changelog.html#airflow-2-2-4-2022-02-22). Afaict there are no SnowflakeOperator related changes but we do need to plan to keep these tests updated over time.

Reply:
Yep - this matrix style tests enables us to easily update airflow versions we're running tests on.

BTW, I've created issue on how to track Airflow (and other systems releases) more reliably - feel free to voice your opinion about this here: https://github.com/OpenLineage/OpenLineage/issues/602

Reply:
I don't understand why you're using this method instead of a `>=2.0.X` so it can test any Airflow 2 version someone is running.